documentation/deliverables/individual_deliverables/sprint_reporting_dashboard.md

```diff
@@ -0,0 +1,264 @@
+- How easy is it to add new charts and metrics?
+- Can users can export or download the data behind a given dashboard?
+- Can system-to-system users access the data via API?
+- Can open source contributors host the dashboard locally?
+- Does the platform used to build the dashboard also support ad hoc data analysis?
+- Can the dashboard include narrative text that explains how the metrics are calculated?
+
+### ETL pipeline
+
+This deliverable should implement a strategy for extracting, transforming, and loading the data needed to calculate the operational metrics being published.
+
+Some options to consider include:
+
+1. GitHub actions + custom scripts
+2. Orchestration tool (e.g. Airflow, Prefect, Dagster, etc.)
+3. SaaS ETL tools (e.g. Talend, Informatica, etc.)
+
+The criteria that should inform this decision include:
+
+- How easy is it to build and maintain new ETL pipelines?
+- How easy is it to connect to new sources or targets?
+- Can the ETL pipeline execute transformations written in different languages (e.g. Python, SQL, Node.js)?
+- What options are supported for scheduling, retries, and dependencies between steps?
+- Can open source contributors host and execute their own version of this pipeline?
+
+### Data warehouse
+
+This deliverable should also implement a strategy for storing the data needed to calculate operational metrics and (potentially) the results of those calculations.
+
+Some options to consider include:
+
+1. Files stored in S3 buckets (e.g. json, csv)
+2. Separate schema in our existing Postgres database
+3. Self-hosted OLAP database (e.g. ClickHouse)
+4. Third-party managed data warehouse (e.g. Snowflake)
+
+The criteria that should inform this decision include:
+
+- Can the warehouse support both structured and unstructured data?
+- How easy is it to query the data for ad hoc reporting and analysis?
+- What kind of access controls does the warehouse provide?
+- Can open source contributors host their own version of this data warehouse?
+
+### Definition of done
+<!-- Required -->
+
+#### Must have
+
+- [ ] Basic requirements
+  - [ ] Code is deployed to main & PROD through our CI/CD pipeline
+  - [ ] Services are live in PROD (may be behind feature flag)
+  - [ ] All new services have passed a security review (as needed)
+  - [ ] All new services have completed a 508 compliance review (as needed)
+  - [ ] Data needed for metrics is actively being captured in PROD
+  - [ ] Key architectural decisions made about this deliverable are documented publicly (as needed)
+- [ ] Functional requirements
+  - [ ] HHS staff can view the dashboard when they are on the HHS network
+  - [ ] Members of the public can view the dashboard without a login
+  - [ ] Members of the public can access the data that populates the dashboard
+  - [ ] The data within the dashboard is refreshed *at least* once per day
+  - [ ] The dashboard displays the following metrics (at a minimum):
```

Comment:
Could we add something simple that connects to the opportunities data? Such as, # of new opportunities posted today? 

And maybe something simple that connects to our Google Analytics data, either in our simpler.grants.gov GA account or the already-publicly-available DAP GA account?

This would help us ensure our architecture is connected to the production database in Simpler environments as well as the Google analytics.

But maybe this squeezes too much into the 30k? We can also defer these until later.

Reply:
I see your comments below about "not doing" - that's fine, we can make these part of a future deliverable then.

Reply:
Yeah I think it would be good to confine this to a single category of operational metrics for this first 30k deliverable. Once we have the basic architecture in place (e.g. get from 0 to 1 with the dashboard) adding new metrics should become much easier.